### PR TITLE
hostname-fix

### PR DESCRIPTION
**Pull request by mridulnagpal**

### CODE REVIEW

File: plinth/modules/config/config.py [hunk 1]
```diff
@@ -250,6 +251,17 @@ def set_hostname(hostname):
                                      old_hostname=old_hostname,
                                      new_hostname=hostname)
 
+    # Domain name should be ASCII. If it's unicode, convert to ASCII.
+    domainname = old_domainname
+
+    LOGGER.info('Changing domain name to - %s', domainname)
+    actions.superuser_run('domainname-change', [domainname])
+
+    domainname_change.send_robust(sender='config',
```
Comment (jvalleroy) [inline plinth/modules/config/config.py]: In this case we are not really changing the domain (it should end up the same before and after). So I don't think we need to send the domainname_change signal.

**Comment by mridulnagpal** [comment]
So what do we do?

**Comment by jvalleroy** [comment]
@mridulnagpal To clarify, I think we can remove the call to domainname_change.send_robust()....lines 260-262.

If you agree that we should remove this, you can make an additional commit and push it to your hostname-domainname-issue branch on github. Then it will show up as part of this pull request.

**Comment by mridulnagpal** [comment]
Done and committed to the PR...

**Comment by jvalleroy** [comment]
Looks good! I've merged this change with minor adjustments.

**Pull request closed without merging**